src/openzeppelin/token/erc721/utils/ERC721Holder.cairo

```diff
@@ -1,14 +1,14 @@
 # SPDX-License-Identifier: MIT
-# OpenZeppelin Contracts for Cairo v0.2.0 (token/erc721/utils/ERC721_Holder.cairo)
+# OpenZeppelin Contracts for Cairo v0.2.0 (token/erc721/utils/ERC721Holder.cairo)
```

Comment:
mh this one is not strictly following the `presets/` directory requirement in CONTRIBUTING. not sure if it needs to, but at least it's worth noticing

Reply:
maybe moving `utils` so it's `erc721/presets/utils` instead? or just removing the directory entirely